src/cfupdater-v4

```diff
@@ -44,4 +44,4 @@ case "$update" in
   exit 1;;
 *)
   echo "[Cloudflare DDNS] IPv4 context '$ip' has been synced to Cloudflare.";;
-esac
\ No newline at end of file
+esac
```

Comment:
EOL missing. Please remove any newline after the end of a file.

Reply:
These are added automatically by github because the industry standard is to always have trailing linebreaks in bash files. these edits were done through the browser editor. I don't have enough invested here to clone this, change my editor settings which also would auto-add trailing newlines, and repush for a  one-line change to using -4/-6 flags. Just trying to help you out with your little code sample

Reply:
No problem! Will get it merged right away!